src/App.elm

```diff
@@ -68,11 +68,17 @@ update msg model =
         FetchMeLoaded result ->
             case result of
                 Ok me ->
-                    ( { model
-                        | loggedIn = LIUpdate.setMe model.loggedIn me
-                      }
-                    , Cmd.none
-                    )
+                    case model.authStatus of
+                        LoggedIn loginInfo ->
+                            ( { model
+                                | loggedIn = LIUpdate.setMe model.loggedIn me
+                              }
+                            , fetchConversations loginInfo me.id
+                                |> Task.perform FetchConversations FetchConversations
+                            )
+
+                        LoggedOut ->
+                            ( model, Cmd.none )
```

Comment:
I'll do it for now, based on the fact that we aren't masking any types as you pointed out with `_` vs. `LoggedOut`.